File: scrape.py
import asyncio
from playwright.async_api import async_playwright

URLS = [
    "https://sanand0.github.io/tdsdata/js_table/?seed=72",
    "https://sanand0.github.io/tdsdata/js_table/?seed=73",
    "https://sanand0.github.io/tdsdata/js_table/?seed=74",
    "https://sanand0.github.io/tdsdata/js_table/?seed=75",
    "https://sanand0.github.io/tdsdata/js_table/?seed=76",
    "https://sanand0.github.io/tdsdata/js_table/?seed=77",
    "https://sanand0.github.io/tdsdata/js_table/?seed=78",
    "https://sanand0.github.io/tdsdata/js_table/?seed=79",
    "https://sanand0.github.io/tdsdata/js_table/?seed=80",
    "https://sanand0.github.io/tdsdata/js_table/?seed=81",
]

async def get_sum(page, url):
    await page.goto(url, wait_until="networkidle")
    cells = await page.query_selector_all("table td, table th")
    total = 0
    for cell in cells:
        text = (await cell.inner_text()).strip()
        try:
            total += float(text)
        except:
            pass
    return total

async def main():
    grand_total = 0
    async with async_playwright() as p:
        browser = await p.chromium.launch()
        page = await browser.new_page()
        for url in URLS:
            s = await get_sum(page, url)
            print(f"Seed {url.split('=')[-1]}: {s}")
            grand_total += s
        await browser.close()
    print(f"Grand Total: {grand_total}")

asyncio.run(main())

File: scrape.js
const { chromium } = require('playwright');

const urls = [
  'https://tools.datasci.study/seed-72/',
  'https://tools.datasci.study/seed-73/',
  'https://tools.datasci.study/seed-74/',
  'https://tools.datasci.study/seed-75/',
  'https://tools.datasci.study/seed-76/',
  'https://tools.datasci.study/seed-77/',
  'https://tools.datasci.study/seed-78/',
  'https://tools.datasci.study/seed-79/',
  'https://tools.datasci.study/seed-80/',
  'https://tools.datasci.study/seed-81/'
];

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  let grandTotal = 0;
  
  for (const url of urls) {
    console.log(`Scraping: ${url}`);
    await page.goto(url, { waitUntil: 'networkidle' });
    const numbers = await page.$$eval('table td, table th', cells => {
      return cells.map(cell => {
        const num = parseFloat(cell.innerText.trim());
        return isNaN(num) ? 0 : num;
      }).filter(n => n !== 0);
    });
    const pageSum = numbers.reduce((a, b) => a + b, 0);
    grandTotal += pageSum;
  }
  
  console.log(`GRAND TOTAL: ${grandTotal}`);
  await browser.close();
})();
